vendingMachine.js

```diff
@@ -13,34 +13,50 @@ let stock = [
 function showVendingMachine(arr1, arr2) {
   let result = '';
   let coord, stoc;
+  let space = " ";
+  let supline = "-";
+  let supli = supline.repeat(122);
+  console.log(supli);

```

Comment:
Is this console needed?

Reply:
yes, because the console prints only once the top line of the table, before starting the for loop.